### PR TITLE
Don't do nasty wrapping of text components - instead, do it nicely

### PR DESCRIPTION
**Pull request by kashike**
nicer alternative to https://github.com/SpigotMC/BungeeCord/commit/fc64a6c2ff64b4cf33c4ba91ae937559bb65cb01

### CODE REVIEW

File: chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java [hunk 1]
```diff
@@ -39,17 +39,12 @@
 
     public static String toString(BaseComponent component)
     {
-        // 1.9 Requires the first component to not be a plain string which can
-        // happen if a text component has no formatting. This optimization is
-        // still useful when nested more so we just manually wrap everything in
-        // an extra text component.
-        return "{\"text\":\"\", \"extra\": [" + gson.toJson( component ) + "]}";
+        return gson.toJson( component );
```
Comment (Black-Hole) [inline chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java]: Ain't you missing the wrapper here?

Comment (kashike) [inline chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java]: Nope. See https://github.com/SpigotMC/BungeeCord/commit/fc64a6c2ff64b4cf33c4ba91ae937559bb65cb01#diff-70a7132b29f11a5540b80443bd73f630R39

**Comment by Thinkofname** [comment]
This was what I was originally going to do but I wasn't sure how bad the loss of that optimization would be. I guess its only 9 bytes extra though

**Comment by kashike** [comment]
Any chance this can be pulled soon @Thinkofname?

**Comment by Thinkofname** [comment]
Sorry, had been busy with Uni work. Pulled with modified commit message here: https://github.com/SpigotMC/BungeeCord/commit/aaddc9fcfdcada8d19ab115f2dcc8aed413a9206

**Pull request closed without merging**